packages/runtime-dom/src/directives/vModel.ts

```diff
@@ -75,7 +75,7 @@ export const vModelText: ModelDirective<
   },
   // set value on mounted so it's after min/max for type="range"
   mounted(el, { value }) {
-    el.value = value == null ? '' : value
+    el.value = value === null ? '' : value
```

Comment:
value == null: value can be `null` or `undefined`
value === null: value must be `null`

Reply:
then (value === null || value === undefined)